Library/Homebrew/cmd/mark.rb

```diff
@@ -0,0 +1,100 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "abstract_command"
+require "formula"
+require "tab"
+
+module Homebrew
+  module Cmd
+    class Mark < AbstractCommand
+      cmd_args do
+        description <<~EOS
+          Mark or unmark <formula> as installed on request, or installed as a
+          dependency.
+        EOS
```

Comment:
Suggestion: It might be helpful to provide a short explanation of why this is useful.